mcp_zammad/server.py

```
@@ -716,18 +1120,54 @@
             except Exception as e:
```

Ruff (0.14.4)

Do not catch blind exception: `Exception`

(BLE001)